apps/api/src/app/chat/chat.config.ts

```diff
@@ -6,7 +6,7 @@ export const assistantParams: AssistantCreateParams = {
   name: '@boldare/openai-assistant',
   instructions: `You are a chatbot assistant. Use the general knowledge to answer questions. Speak briefly and clearly.`,
   tools: [{ type: 'code_interpreter' }, { type: 'file_search' }],
-  model: 'gpt-4o',
+  model: 'gpt-4o-mini',
```

Comment:
Czy chcemy zmieniać model domyślnie?

Reply:
yes, in this case, we can reduce costs and this model is enough for our test cases for our demo